MobileCenterPush/MobileCenterPush.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,745 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXAggregateTarget section */
+		D3AD5F221E5C9562001627AB /* MobileCenterPushFramework */ = {
+			isa = PBXAggregateTarget;
+			buildConfigurationList = D3AD5F251E5C9562001627AB /* Build configuration list for PBXAggregateTarget "MobileCenterPushFramework" */;
+			buildPhases = (
+				D3AD5F261E5C9588001627AB /* ShellScript */,
+			);
+			dependencies = (
+			);
+			name = MobileCenterPushFramework;
+			productName = MobileCenterPushFramework;
```

Comment:
I have no idea why but you need to change this productName to "Fat Framework" to have all consistent names across all projects.